src/snowflake/cli/_plugins/nativeapp/sf_sql_facade.py

```diff
@@ -254,15 +255,15 @@ def get_account_event_table(self, role: str | None = None) -> str | None:
     def create_version_in_package(
         self,
         package_name: str,
-        stage_fqn: str,
+        path_to_version_directory: str,
```

Comment:
i am confused about the naming here. Why we mention version_directory here (while the rest uses stage_subdirectory), and doesn't this also include stage info? Is stage_fqn not accurate anymore? Or stage_with_subdirectory? or StageParts class?

Reply:
or stage_full_path

Reply:
path_to_version_directory is the naming used in the official docs for these two operations, so the facade naming is matching that.